jenkins/test-history/gen_html.py

```diff
@@ -45,6 +46,38 @@
 ])
 
 
+PREFIX_TO_URL = {}
+
+def prefix_for(prefix):
+    if 'azure' in prefix:
+        new = 'azure$'
+    elif 'rktnetes' in prefix:
+        new = 'rktnetes$'
+    elif 'gs://kubernetes-jenkins/' in prefix:
+        new = ''
+    else:
+        new = prefix[5:-1].replace('/', '_') + '$'
+    PREFIX_TO_URL[new] = prefix
+    return new
+
+
+def slugify(inp):
```

Comment:
Please add a short comment explaining what this does. Even though it's so short, it's pretty dense regex magic :P